frontend/src/worksheets/TableListOfWorkSheets.js

```diff
@@ -42,6 +52,13 @@ export default function TableListOfWorkSheets() {
                     <td>{worksheet.partnerId}</td>
                     <td>{typeOfWork[worksheet.typeOfWork]}</td>
                     <td>{status[worksheet.worksheetStatus]}</td>
+                    <td>
+                      {isAdmin && worksheet.worksheetStatus !== 'CLOSED' && (
```

Comment:
mindig van beállítva `worksheetStatus` attributuma egy worksheetnek? Vagy ez nem annyire releváns?
Jelen esetben bármilyen más értéke is van (pl OPEN, null, "", undefined) akkor is le lehet zárni... okozhat ez bajt?

Reply:
Elvileg van mindig, back enden a worksheet status egy enum, 3 féle értéke lehet.